rfcs/20200712-tfrt-kernel-fallback.md

```diff
@@ -0,0 +1,647 @@
+# TFRT Kernel Fallback
+
+| Status        | (Proposed / Accepted / Implemented / Obsolete)          |
+| :------------ | :------------------------------------------------------ |
+| **RFC #**     | [NNN](https://github.com/tensorflow/community/pull/NNN) |
+| **Author(s)** | Anna Revinskaya (annarev@google.com), Jeremy Lau (lauj@google.com) |
+| **Sponsor**   | Jeremy Lau (lauj@google.com)                            |
+| **Updated**   | 2020-07-14                                              |
+
+## Objective
+
+This proposal focuses on getting majority of “well-behaved” ops running in
+[TF Lite](https://www.tensorflow.org/lite) by skipping current eager runtime and
+calling kernels directly from [TFRT](https://github.com/tensorflow/runtime) (a new
+TensorFlow runtime).
```

Comment:
This wording is a bit confusing.
Maybe, ....

"This proposal focuses on getting a majority of "well-behaved" TensorFlow ops running efficiently on
mobile devices by removing the need to execute them via the TensorFlow eager runtime, instead
calling kernels directly from the new [TFRT](https://github.com/tensorflow/runtime) TensorFlow runtime."

Reply:
Updated.